library/formatting.inc.php

```diff
@@ -74,6 +74,44 @@ function oeFormatTime($time, $format = "")
     return $formatted;
 }
 
+/**
+ * Returns the complete formatted datetime string according the global date and time format
+ * @param $datetime
+ * @return string
+ */
+function oeFormatDateTime($datetime)
```

Comment:
also added this function which will do the same thing without requiring a timestamp (uses datetime as input), which will work in cases when is prior to 1970 or after 2038.